<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="AMICHAITO1-HHS" ignored="5" name="Test-class test 1 red0years" tests="8" failures="7" timestamp="22 Jan 2020 09:42:23 GMT" time="222.876" errors="0">
  <testcase name="initialOfferQuestions" time="59.7" classname="TestNG.sanityAllOptionsTest"/>
  <testcase name="initialOfferScreen" time="97.089" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in INITIAL OFFER SCREEN QUESTION">
      <![CDATA[java.lang.Exception: Failed in INITIAL OFFER SCREEN QUESTION
at TestNG.sanityAllOptionsTest.initialOfferScreen(sanityAllOptionsTest.java:531)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- initialOfferScreen -->
  <testcase name="finalOfferQuestions" time="10.007" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in ScreenShot">
      <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.emailException(sanityAllOptionsTest.java:108)
at TestNG.sanityAllOptionsTest.finalOfferQuestions(sanityAllOptionsTest.java:766)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- finalOfferQuestions -->
  <testcase name="finalOfferScreen" time="10.002" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in ScreenShot">
      <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.emailException(sanityAllOptionsTest.java:108)
at TestNG.sanityAllOptionsTest.finalOfferScreen(sanityAllOptionsTest.java:828)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- finalOfferScreen -->
  <testcase name="covers" time="10.007" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in ScreenShot">
      <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.emailException(sanityAllOptionsTest.java:108)
at TestNG.sanityAllOptionsTest.covers(sanityAllOptionsTest.java:904)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- covers -->
  <testcase name="payments" time="13.007" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in ScreenShot">
      <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.emailException(sanityAllOptionsTest.java:108)
at TestNG.sanityAllOptionsTest.payments(sanityAllOptionsTest.java:948)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- payments -->
  <testcase name="returnSteps" time="10.006" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in ScreenShot">
      <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.emailException(sanityAllOptionsTest.java:108)
at TestNG.sanityAllOptionsTest.returnSteps(sanityAllOptionsTest.java:1076)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- returnSteps -->
  <testcase name="forwardSteps" time="13.003" classname="TestNG.sanityAllOptionsTest">
    <failure type="java.lang.Exception" message="Failed in ScreenShot">
      <![CDATA[java.lang.Exception: Failed in ScreenShot
at TestNG.sanityAllOptionsTest.emailException(sanityAllOptionsTest.java:108)
at TestNG.sanityAllOptionsTest.forwardSteps(sanityAllOptionsTest.java:1185)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- forwardSteps -->
  <testcase name="callMeWizard2" time="0.0" classname="TestNG.sanityAllOptionsTest">
    <ignored/>
  </testcase> <!-- callMeWizard2 -->
  <testcase name="callMeWizard1" time="0.0" classname="TestNG.sanityAllOptionsTest">
    <ignored/>
  </testcase> <!-- callMeWizard1 -->
  <testcase name="thankYouPage" time="0.0" classname="TestNG.sanityAllOptionsTest">
    <ignored/>
  </testcase> <!-- thankYouPage -->
  <testcase name="registeredAccount" time="0.0" classname="TestNG.sanityAllOptionsTest">
    <ignored/>
  </testcase> <!-- registeredAccount -->
  <testcase name="iframeCreditGuard" time="0.0" classname="TestNG.sanityAllOptionsTest">
    <ignored/>
  </testcase> <!-- iframeCreditGuard -->
</testsuite> <!-- Test-class test 1 red0years -->
